### PR TITLE
js: eshost-support related fixes

### PR DESCRIPTION
**Pull request by IdanHo**

### CODE REVIEW

File: Userland/Utilities/js.cpp [hunk 1]
```diff
@@ -738,7 +738,8 @@ static void print_value(JS::Value value, HashTable<JS::Object*>& seen_objects)
             return print_function(object, seen_objects);
         if (is<JS::Date>(object))
             return print_date(object, seen_objects);
-        if (is<JS::Error>(object))
+        auto prototype_or_error = object.internal_get_prototype_of();
+        if (is<JS::Error>(object) || (prototype_or_error.has_value() && prototype_or_error.value() == object.global_object().error_prototype()))
```
Comment (linusg) [inline Userland/Utilities/js.cpp:743]: `print_object` at the end might crash if we call it with a stored exception - also, can we only do the prototype lookup for non-`JS::Error` objects?

**Comment by ADKaster** [comment]
Should we add an install rule for `js` in CMake lists? I noticed running some stuff this weekend that ninja install on the Lagom directory doesn't put the utilities in lagom-install/bin

File: Userland/Utilities/js.cpp (hunk 1 above)
Comment (IdanHo) [reply Userland/Utilities/js.cpp]: ugh, I keep forgetting that we still throw normal exceptions even with ThrowCompletionOr, fixed.

**Comment by IdanHo** [comment]
> Should we add an install rule for `js` in CMake lists? I noticed running some stuff this weekend that ninja install on the Lagom directory doesn't put the utilities in lagom-install/bin

That would probably be nice, yeah

**Comment by linusg** [comment]
Code LGTM, but almost forgot about the man page :)
https://github.com/SerenityOS/serenity/blob/master/Base/usr/share/man/man1/js.md

**Comment by IdanHo** [comment]
> Code LGTM, but almost forgot about the man page :) [`master`/Base/usr/share/man/man1/js.md](https://github.com/SerenityOS/serenity/blob/master/Base/usr/share/man/man1/js.md?rgh-link-date=2021-11-24T14%3A41%3A58Z)

Whoops! Fixed. (and also added a commit that adds a couple of other js options that were already missing)

**Review by linusg — APPROVED** [review]

**Pull request merged**